### PR TITLE
Update gmt usage message

### PR DESCRIPTION
**Pull request by PaulWessel**
Instead of displaying an ever-increasing list of core developers, just refer to the GMT team + URL.

### CODE REVIEW

File: src/gmt.c [hunk 1]
```diff
@@ -220,15 +220,13 @@ int main (int argc, char *argv[]) {
...
-			fprintf (stderr, "\n\t  GMT - The Generic Mapping Tools, Version %s [%u cores]\n", GMT_VERSION, api_ctrl->n_cores);
-			fprintf (stderr, "(c) 1991-%d Paul Wessel, Walter H. F. Smith, Remko Scharroo, Joaquim Luis, and Florian Wobbe\n\n", GMT_VERSION_YEAR);
+			fprintf (stderr, "\n\tGMT - The Generic Mapping Tools, Version %s [%u cores]\n", GMT_VERSION, api_ctrl->n_cores);
+			fprintf (stderr, "\t(c) 1991-%d The GMT Team (https://www.generic-mapping-tools.org/team.html).\n\n", GMT_VERSION_YEAR);
 			fprintf (stderr, "Supported in part by the US National Science Foundation (http://www.nsf.gov/)\n");
 			fprintf (stderr, "and volunteers from around the world (see http://gmt.soest.hawaii.edu/).\n\n");
 
-			fprintf (stderr, "This program comes with NO WARRANTY, to the extent permitted by law.\n");
+			fprintf (stderr, "GMT is distributed under the GNU LGP License (http://www.gnu.org/licenses/lgpl.html).\n");
 			fprintf (stderr, "You may redistribute copies of this program under the terms of the\n");
```
Comment (seisman) [inline src/gmt.c]: Line 229 should be removed.

**Comment by seisman** [comment]
This is what I get:
```

	GMT - The Generic Mapping Tools, Version 6.0.0_6306303_2019.06.11 [64-bit] [8 cores]
	(c) 1991-2019 The GMT Team (https://www.generic-mapping-tools.org/team.html).

Supported in part by the US National Science Foundation (http://www.nsf.gov/)
and volunteers from around the world (see http://gmt.soest.hawaii.edu/).

GMT is distributed under the GNU LGP License (http://www.gnu.org/licenses/lgpl.html).
usage: gmt [options]
       gmt <module name> [<module-options>]

session management:
  gmt begin         Initiate a new GMT session using modern mode [classic].
  gmt end           Terminate the current GMT modern mode session.
  gmt docs          Display the HTML documentation for selected module.
  gmt figure        Set figure format specifics under a GMT modern mode session.
  gmt subplot       Initiate a multi-panel figure.
  gmt clear all | cache | cpt | conf | data | history | sessions
                    Delete gmt.history, gmt.conf, session CPT, the user cache or data dir, or all of them.
```

Issues:

1. Need to add a blank line between license and usage.
2. `gmt clear` is a little too long. Can you make it shorter?

**Comment by joa-quim** [comment]
> and volunteers from around the world (see http://gmt.soest.hawaii.edu/).

We are abandoning the ``soest`` addresses right. Maybe drop this line all

**Comment by PaulWessel** [comment]
I replaced the Wessel, Smith, etc Copyright with the GMT Team (URL) and the Contact info at soest with generic-mapping-tools, and a few similar changes throughout the repo.  There are a few remaining references to soest.hawaii.edu, particularly in the *.md files since that is in fact where the tar balls etc can be found.  We may need to change this soon.

File: doc/rst/source/gmt.rst [hunk 2]
```diff
@@ -260,4 +259,4 @@ Look up the individual man pages for more details and full syntax. Run
 directories. For an explanation of the various GMT settings in this
 man page (like :ref:`FORMAT_FLOAT_OUT <FORMAT_FLOAT_OUT>`), see the man page of the GMT
 configuration file :doc:`gmt.conf`. Information is also available on the
-GMT home page http://gmt.soest.hawaii.edu/
+GMT home page http://docs.generic-mapping-tools.org/
```
Comment (seisman) [inline doc/rst/source/gmt.rst]:
```suggestion
GMT documentation site https://docs.generic-mapping-tools.org/
```

File: doc/rst/source/gmt_shell_functions.sh.rst [hunk 3]
```diff
@@ -176,7 +176,7 @@ functions do not work in the C shell (**csh**) or their derivatives
 2. **gmt_shell_functions.sh** were first introduced in GMT version
 4.2.2 and have since been regularly expanded with other practical
 scripting short-cuts. If you want to suggest other functions, please do
-so by adding a New Issue request on gmt.soest.hawaii.edu.
+so by adding a New Issue request on www.generic-mapping-tools.org.
```
Comment (seisman) [inline doc/rst/source/gmt_shell_functions.sh.rst]:
```suggestion
so by adding a New Issue request on https://github.com/GenericMappingTools/gmt.
```

File: src/gmt.c [hunk 4]
```diff
@@ -220,15 +220,12 @@ int main (int argc, char *argv[]) {
...
+			fprintf (stderr, "\t(c) 1991-%d The GMT Team (https://www.generic-mapping-tools.org/team.html).\n\n", GMT_VERSION_YEAR);
 			fprintf (stderr, "Supported in part by the US National Science Foundation (http://www.nsf.gov/)\n");
-			fprintf (stderr, "and volunteers from around the world (see http://gmt.soest.hawaii.edu/).\n\n");
+			fprintf (stderr, "and volunteers from around the world.\n\n");
 
-			fprintf (stderr, "This program comes with NO WARRANTY, to the extent permitted by law.\n");
-			fprintf (stderr, "You may redistribute copies of this program under the terms of the\n");
-			fprintf (stderr, "GNU Lesser General Public License (http://www.gnu.org/licenses/lgpl.html).\n");
-			fprintf (stderr, "For more information about legal matters, see the file named LICENSE.TXT.\n\n");
+			fprintf (stderr, "GMT is distributed under the GNU LGP License (http://www.gnu.org/licenses/lgpl.html).\n");
```
Comment (seisman) [inline src/gmt.c]:
```suggestion
			fprintf (stderr, "GMT is distributed under the GNU LGP License (http://www.gnu.org/licenses/lgpl.html).\n\n");
```

**Comment by PaulWessel** [comment]
Any objections to removing WIP from this an committing?

**Comment by seisman** [comment]
I just made a few changes. Besides that, `the GNU LGP License` seems a little odd to me. Usually it's called GNU LGPL or GNU Lesser GPL.

**Pull request merged**